.ruff.toml

```diff
@@ -265,6 +266,7 @@ select = [
     "S609",  # Possible wildcard injection in call due to `*` usage
     "S612",  # Use of insecure `logging.config.listen` detected
 #    "S701",  # Using jinja2 templates with `autoescape=False` is dangerous and can lead to XSS. Ensure `autoescape=True` or use the `select_autoescape` function.
+    "S702",  # Mako templates allow HTML and JavaScript rendering by default and are inherently open to XSS attacks
```

Comment:
I don't think we use Mako templates (unless you wanted to say "Make")

Reply:
This file `.ruff.toml` contains a complete list of rules for each category of rules. The rules in this PR are the only missing rules, and since they all are new rules, I presume they should be added for completeness and consistency. Unwanted or unused rules are commented out.

Are you suggesting commenting this rule out?

Reply:
I think we can comment it out? or maybe add a comment saying it's unused but keep it in case we use it but forget about enabling it?

Reply:
I have commented it just like the above rule "S701".